lib/src/main/java/de/siegmar/fastcsv/util/Limits.java

```diff
@@ -27,4 +34,24 @@ public final class Limits {
     private Limits() {
     }
 
+    /**
+     * Retrieves the system property value as an integer, with a fallback to a default value.
+     * If the property is not set or cannot be parsed, the default value is returned.
```

Comment:
The implementation does not use the default value if it cannot be parsed (which is good IMO). The same applies to the documentation of the defaultValue and return value.